src/lib.rs

```diff
@@ -76,9 +73,18 @@ pub async fn app(config: &Figment) -> Result<Rocket<Build>> {
     tracing::tracing_try_init(&kepler_config.log);
 
     storage::KV::healthcheck(kepler_config.storage.indexes.clone()).await?;
-    let kp = kepler_config.storage.blocks.relay_key_pair().await?;
 
-    let relay_node = RelayNode::new(kepler_config.relay.port, Keypair::Ed25519(kp)).await?;
+    let mut relay_node = RelayConfig::default().launch(
+        kepler_config.storage.blocks.relay_key_pair().await?,
+        Both::<MemoryConfig, TcpConfig>::default(),
+    )?;
+
+    relay_node
+        .listen_on([
+            build_multiaddr!(Memory(kepler_config.relay.port)),
+            build_multiaddr!(Ip4([127, 0, 0, 1]), Tcp(kepler_config.relay.port)),
```

Comment:
I've changed the relay address config to be a multi address, so it would go from
```
address = "127.0.0.1"
port = "8081"
```
to
```
address = "/ip4/127.0.0.1/tcp/8081"
```
or any other protocols supported by libp2p

Reply:
That was reverted because those variables are picked up by Rocket (see other thread)